src/contracts/atlas/AtlasVerification.sol

```diff
@@ -117,7 +118,7 @@ contract AtlasVerification is EIP712, DAppIntegration, AtlasConstants {
             }
 
             // Check user signature
-            ValidCallsResult verifyUserResult = _verifyUser(dConfig, userOp, msgSender, isSimulation);
+            ValidCallsResult verifyUserResult = _verifyUser(dConfig, userOp, userOpHash, msgSender, isSimulation);
```

Comment:
```suggestion
            ValidCallsResult verifyUserResult = _verifyUser(dConfig, userOp, msgSender, isSimulation);
```
Remove userOpHash here and in function definition below if not used as per @jj1980a's note